packages/wrangler/src/__tests__/pages.test.ts

```diff
@@ -751,4 +740,273 @@ describe("pages", () => {
       expect(std.err).toMatchInlineSnapshot(`""`);
     });
   });
+
```

Comment:
I'm not a big fan of duplicating tests since the next person won't know why they're both there and end up updating them both. If they're really the same test case for two different commands, I'd use `describe.each` (https://jestjs.io/docs/api#describeeachtablename-fn-timeout) and leave a comment as to what the two commands have to do with each other.

Or are the two sets of tests a bit more different than I'm thinking?

Reply:
> I've copy-pasted the existing tests for the time being, but we could probably remove them from the 'pages publish' testing block in the future.

While this is still being worked on, I'm just going to leave this as-is. I'm pretty sure we'll have some follow-up PRs for this work, so I want to just come back to it, once we're happy with the plan for how things work.